sycl/include/sycl/ext/oneapi/experimental/device_architecture.hpp

```diff
@@ -21,7 +21,7 @@ enum class architecture {
   intel_gpu_tgllp,
   intel_gpu_rkl,
   intel_gpu_adl_s,
-  intel_gpu_rpl_s,
+  intel_gpu_rpl_s = intel_gpu_adl_s,
```

Comment:
Don't you also need to change `MapDeviceIpToArch` in "./source/detail/device_info.hpp"?